CMakeLists.txt

```diff
@@ -63,9 +64,18 @@ add_definitions("-DGIT_COMMIT_TAG=\"${GIT_COMMIT_TAG}\"")
 add_definitions(-std=c99)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_GNU_SOURCE -DNOPOLL_LOGGER ")
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror -Wall -Wno-missing-field-initializers")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror -Wall")
-
+if (DEVICE_CAMERA)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-error=all -Wno-missing-field-initializers")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=all")
+add_definitions(-DDEVICE_CAMERA)
+else (DEVICE_CAMERA)
```

Comment:
Can you remove "(DEVICE_CAMERA)" word from this else case as it get confused easily , no harm or logic change but for better readability for other platforms .

Reply:
Removed text "DEVICE_CAMERA" from the else case